source/LumiCalReco/src/LumiCalClusterer_getCalHits.cpp

```diff
@@ -78,7 +78,9 @@ int LumiCalClustererClass::getCalHits(	EVENT::LCEvent * evt,
 
       } else {
 	arm =(*_mydecoder)( calHitIn )["barrel"]; // from 1 and 2
-	phiCell = (*_mydecoder)( calHitIn )["phi"];
+	//FIXME: add check for phi range?
+	phiCell = (*_mydecoder)( calHitIn )["phi"]; // goes from -phiMax/2 to +phiMax/2
+	if(phiCell < 0 ) phiCell += _cellPhiMax; // need to put into positive range only
```

Comment:
Yes, negative phiCellID can be expected. 
In principle the phiOffset could be set for the segmentation to make phiCellIDs be always positive. The best would be to solve this for all cases though.